src/gam/gapi/directory/printers.py

```diff
@@ -76,15 +58,32 @@ def create():
 
 
 def delete():
-    '''gam delete printer'''
+    '''gam delete printer <PrinterID>|(crosfile <FileName>)|(croscsvfile <FileName>:<FieldName>)'''
     cdapi = gapi_directory.build()
     customer_id = _get_customerid()
     printer_id = sys.argv[3]
-    name = f'{customer_id}/chrome/printers/{printer_id}'
-    gapi.call(cdapi.customers().chrome().printers(),
-                       'delete',
-                       name=name)
-    print(f'Deleted printer {printer_id}')
+    if printer_id.lower() not in {'crosfile', 'croscsvfile'}:
+        name = f'{customer_id}/chrome/printers/{printer_id}'
+        gapi.call(cdapi.customers().chrome().printers(),
```

Comment:
this is good but I don't think there's any harm in "batch deleting" 1 printer. Let's just let a list of one printer id fall through to the batch delete call.